pallets/capacity/src/lib.rs

```diff
@@ -512,7 +516,9 @@ impl<T: Config> Pallet<T> {
 
 	/// Calculates Capacity.
 	fn calculate_capacity(amount: BalanceOf<T>) -> BalanceOf<T> {
-		amount
+		let result = Perbill::from_rational(1u32, T::TokenPerCapacity::get().into());
```

Comment:
Setting it as a fraction in the Config would remove this.

Reply:
Do you mean storing it as a float, or as two numbers?